Interview and introduction (#68)
WhitakerLab/whitakerlab.github.io: _posts/blog/2019-06-20-Hello!I am Yini He.md

> ---

Reviewer: This is looking great @minnieho1115! Just a few comments and then we should be ok to go.

Please can you rename the file without the `!`? I think it will confuse some websites to have it in the filename.

Author: Thank Kirstie for giving me lots of great comments!

layout: post

> title: Hello:)I am Yini He!

Reviewer: ```suggestion
title: Hello :) I am Yini He!
```

## Who am I

+>I am who I am（我就是我）

My name is Yini He（何旖旎）.
I’m a PhD student with a major in neuroimaging at the University of Electronic Science and Technology of China (UESTC).
It is truly amazing for me to study in the WhitakerLab at the Alan Turing Institute as a visiting student,jointly funded by the Chinese Scholarship Council (CSC) and the Newton Fund.
I am enthusiastic about the relationship between individual traits, the brain and emotions, and particularly the associated risks of developing mental disorders.

## Where I come from  

+>As different as night and day（冰火两重天）

I was born in Changsha city in China, which is famous for the spicy dishes and excessively distinct seasons.
Then I moved to Haikou, which is famous for the gentlest sea wind and spring-like seasons. 
As its title indicates, these two cities have extreme differences, just like a song changing from a gentle and poetic melody to passionate and fiery bursts, fully influencing the two sides of my personality. 
I am happy and passionate at work, and at the same time I enjoy a totally peaceful and quiet life when I'm not working. 

<figure>
  <img src="/images/YiniHe/1coconut5.jpg">

>   <figcaption> Cold drinks and spicy food are best partners!!!(The source of picture is come from internet) </figcaption>

Reviewer: Add link to source.

Author: This is composite diagram I made with Internet resources,hence there is no direct web source.

Reviewer: You can link to the individual pictures you've used.

</figure>

## What are my hobbies?

+>The more National, the more International.（民族的就是世界的）

I love travelling around the world. 
The coexistence of classic culture with the modernity and prosperity of London attracts me incredibly. 
The people here are gentle and full of artistic temperament.  
Modern and beautiful Vancouver attracts me just as much.  
The people there understand the beauty of cultural integration.  
Da Nang beckons to me with its mysteriousness and simplicity. 
The people there are warm and kind. 

<figure>
  <img src="/images/YiniHe/2panda5.jpg">
  <figcaption> The panda truly loves the global village(photo by Yini) </figcaption>
</figure>
                  

> At the same time, I am interested in all kinds of traditional Chinese culture, with its food, traditional clothing, calligaraphy, tea and Guqin(an ancient instrument of China). With a history of 5000 years, China has preserved its ancient philosophy and beautiful civilization. I am fascinated by the merging of Chinese and western cultures. I used to play the ukelele wearing Qipao(a traditional dress of China).  I also tried cooking a Chinese meal with fish and chips. 

Reviewer: Sentences on separate lines.

Author: Thank you.I will do it.

<figure>
  <img src="/images/YiniHe/3qipao5.jpg">

>   <figcaption> One of the most beautiful nouns in the world is culture(photo by Yini) </figcaption>

Reviewer: Watch for missing spaces between words.

```suggestion
  <figcaption> One of the most beautiful nouns in the world is culture (photo by Yini) </figcaption>
```

</figure>

## Life and work goals

+>Ideal comes from life（理想源于生活）

My university major is psychology.  
In my senior year, I decided to go to a special school and work as an intern teacher there.  
The students at this school had mental health differences. 
This was the first time that I truly understood the importance of mental health at heart.  
I loved those children. Many of them had great artistic talents.  
Some of them had attractive and friendly dispositions, and were willing to be best friends with me. 
At the same time I was distressed to think that they could face social difficulties and social burdens. 
Ever since then, I’ve always thought about these teenagers and hoped that they could have a better life with the help of medical and therapeutic advancements for mental health. 

<figure>

>   <img src="/images/YiniHe/4yuanmingyuan!_5.jpg">

Reviewer: Can you save this file name without the `!_`? Just rename to `4yuanmingyuan5.jpg` because it makes the formatting in markdown rather strange to have the `_` and the `!` in the file name.

Author: Thank you for reminding me this good  habit.

  <figcaption> Could we replace tradge with love?(photo by Yini) </figcaption>
</figure>

## My research experience

+Look at the fun of work, so we have fun every day(快乐生活，快乐工作)

> Ever since my experience as an intern at the school for special children, I have been concerned about susceptibility to mental illness.  

Reviewer: ```suggestion
Ever since my experience as an intern at the school for special children, I have been concerned about susceptibility to mental health disorders.  
```

My major was modified to Neuroimaging.  
The brilliance of how the brain works totally amazed me. 
I became very interested in regions of the brain and the connectivity and the fiber bundle of the brain where the source for the link between human behavior and emotion could be found and studied.

> I really appreciate that I have a great supervisor who is Tianzi Jiang.  

Reviewer: Add link to Tianzi Jiang webpage?

Author: Good Idea.Thanks!
